app/(manager)/inventory.tsx

```diff
@@ -76,7 +77,7 @@ export default function InventoryManagementScreen() {
       );
 
       setInventory(updatedInventory);
-      sessionStorage.setItem('ShopInventory', JSON.stringify(updatedInventory));
+      AsyncStorage.setItem('ShopInventory', JSON.stringify(updatedInventory));
```

Comment:
why is this in the session? we don't need to save it that way, unsafe

Reply:
why not it for less read from the db